media/jui/js/jquery.ui.core.js

```diff
@@ -1107,7 +1110,12 @@ $.fn.position = function( options ) {
 	options = $.extend( {}, options );
 
 	var atOffset, targetWidth, targetHeight, targetOffset, basePosition,
-		target = $( options.of ),
+
+		// Make sure string options are treated as CSS selectors - Backport for CVE-2021-41184
```

Comment:
```suggestion
		// Backport for CVE-2021-41184 - Make sure string options are treated as CSS selectors
```

Frontload the text to make it easier to spot when reading the code